src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -880,11 +894,11 @@ export class OpenSheetMusicDisplay {
     public setDrawBoundingBox(value: string, render: boolean = false): void {
         this.drawBoundingBox = value;
         if (this.drawer) {
-            this.drawer.drawableBoundingBoxElement = value; // drawer is sometimes created anew, losing this value, so it's saved in OSMD now.
+        this.drawer.drawableBoundingBoxElement = value; // drawer is sometimes created anew, losing this value, so it's saved in OSMD now.
```

Comment:
fix indentation. your IDE or linter should catch this style issue.

Reply:
my IDE Visual Studio Code applies the eslint configuration in the project. It seems that this is not configured. Done